Auto update failing for vulkan-validation-layers

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [vulkan-validation-layers](https://github.com/termux/termux-packages/blob/ecdefdfd2a4a301a1d10fb5ea287d3e309d66d49/packages/vulkan-validation-layers) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating vulkan-validation-layers [Current version: 1.4.315]
INFO: package being updated to 1.4.316.
Downloading https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.4.316.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 13150    0 13150    0     0  67009      0 --:--:-- --:--:-- --:--:-- 67009
100 6229k    0 6229k    0     0  6088k      0 --:--:--  0:00:01 --:--:-- 7525k

You are about to commit these changes:

--------------------
diff --git a/packages/vulkan-validation-layers/build.sh b/packages/vulkan-validation-layers/build.sh
index 69e22ec70c..acf300343c 100644
--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.4.315"
+TERMUX_PKG_VERSION="1.4.316"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=9e85fcd793fb1ee871ea8e293cb1f9736473614be23dd03a0dec8a5d1cee918c
+TERMUX_PKG_SHA256=afd7baf50d20da649a6b1fac2e291fa598e574a1c10526e50214557a98608644
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
--------------------

bump(main/vulkan-validation-layers): 1.4.316


Not committing to Git!

INFO: Trying to build package.
INFO: Package vulkan-validation-layers exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building vulkan-validation-layers for arch aarch64...
::group::INFO: Fetching repo metadata
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100 13134  100 13134    0     0  82677      0 --:--:-- --:--:-- --:--:-- 83126
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   5513      0 --:--:-- --:--:-- --:--:--  5516
[packages-cf.termux.dev-apt-termux-main]: gpg: Signature made Sat 31 May 2025 12:38:45 AM UTC
[packages-cf.termux.dev-apt-termux-main]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-main]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  81823      0 --:--:-- --:--:-- --:--:-- 82000
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   5709      0 --:--:-- --:--:-- --:--:--  5744
[packages-cf.termux.dev-apt-termux-x11]: gpg: Signature made Fri 30 May 2025 06:43:28 PM UTC
[packages-cf.termux.dev-apt-termux-x11]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-x11]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100 13320  100 13320    0     0  92460      0 --:--:-- --:--:-- --:--:-- 93146
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   5466      0 --:--:-- --:--:-- --:--:--  5480
[packages-cf.termux.dev-apt-termux-root]: gpg: Signature made Fri 23 May 2025 06:24:03 PM UTC
[packages-cf.termux.dev-apt-termux-root]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-root]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
::endgroup::
Downloading dependency libc++@27c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.7.1 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libe/libexpat/libexpat_2.7.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 87760  100 87760    0     0  1961k      0 --:--:-- --:--:-- --:--:-- 1993k
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.7 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libf/libffi/libffi_3.4.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30588  100 30588    0     0   733k      0 --:--:-- --:--:-- --:--:--  746k
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.18 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libiconv/libiconv_1.18_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  548k  100  548k    0     0  10.9M      0 --:--:-- --:--:-- --:--:-- 10.9M
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.18 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/i/iconv/iconv_1.18_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 10020  100 10020    0     0   233k      0 --:--:-- --:--:-- --:--:--  238k
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.8.1 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblzma/liblzma_5.8.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  187k  100  187k    0     0  3902k      0 --:--:-- --:--:-- --:--:-- 3983k
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.8.1 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xz-utils/xz-utils_5.8.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 71040  100 71040    0     0  1731k      0 --:--:-- --:--:-- --:--:-- 1778k
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.25 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gdbm/gdbm_1.25_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  146k  100  146k    0     0  3423k      0 --:--:-- --:--:-- --:--:-- 3482k
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-posix-semaphore/libandroid-posix-semaphore_0.1-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  4128  100  4128    0     0    98k      0 --:--:-- --:--:-- --:--:--  100k
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libb/libbz2/libbz2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 26634  100 26634    0     0   571k      0 --:--:-- --:--:-- --:--:--  577k
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/bzip2/bzip2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 25506  100 25506    0     0   481k      0 --:--:-- --:--:-- --:--:--  488k
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2025.02.25 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2025.02.25 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.4.1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.4.1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libcrypt/libcrypt_0.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8720  100  8720    0     0   221k      0 --:--:-- --:--:-- --:--:--  224k
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/tcl/tcl_8.6.14_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 2209k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2209k  100 2209k    0     0  33.8M      0 --:--:-- --:--:-- --:--:-- 33.7M
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.49.2 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libsqlite/libsqlite_3.49.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  617k  100  617k    0     0  11.6M      0 --:--:-- --:--:-- --:--:-- 11.8M
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.49.2 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/sqlite/sqlite_3.49.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  583k  100  583k    0     0  11.1M      0 --:--:-- --:--:-- --:--:-- 11.1M
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.49.2 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libsqlite-tcl/libsqlite-tcl_3.49.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 17780  100 17780    0     0   362k      0 --:--:-- --:--:-- --:--:--  369k
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0-2 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/brotli/brotli_1.1.0-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  330k  100  330k    0     0  5139k      0 --:--:-- --:--:-- --:--:-- 5163k
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.48 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libp/libpng/libpng_1.6.48_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  190k  100  190k    0     0  3278k      0 --:--:-- --:--:-- --:--:-- 3291k
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/freetype/freetype_2.13.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  399k  100  399k    0     0  6849k      0 --:--:-- --:--:-- --:--:-- 6896k
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/ttf-dejavu/ttf-dejavu_2.37-8_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2410k  100 2410k    0     0  38.5M      0 --:--:-- --:--:-- --:--:-- 38.5M
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.16.2 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fontconfig/fontconfig_2.16.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  119k  100  119k    0     0  2372k      0 --:--:-- --:--:-- --:--:-- 2398k
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.16.2 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fontconfig-utils/fontconfig-utils_2.16.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 14432  100 14432    0     0   303k      0 --:--:-- --:--:-- --:--:--  306k
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.2 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xorg-util-macros/xorg-util-macros_1.20.2_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 21948  100 21948    0     0   416k      0 --:--:-- --:--:-- --:--:--  420k
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xorgproto/xorgproto_2024.1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  243k  100  243k    0     0  4730k      0 --:--:-- --:--:-- --:--:-- 4777k
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.12 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxau/libxau_1.0.12_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  9120  100  9120    0     0   179k      0 --:--:-- --:--:-- --:--:--  181k
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxdmcp/libxdmcp_1.1.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 24352  100 24352    0     0   436k      0 --:--:-- --:--:-- --:--:--  440k
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0-1 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xcb-proto/xcb-proto_1.17.0-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 78616  100 78616    0     0  1471k      0 --:--:-- --:--:-- --:--:-- 1476k
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0-1 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-xcbgen/python-xcbgen_1.17.0-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 16 44572   16  7556    0     0   141k      0 --:--:-- --:--:-- --:--:--  139k
100 44572  100 44572    0     0   825k      0 --:--:-- --:--:-- --:--:--  821k
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxcb/libxcb_1.17.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  913k  100  913k    0     0  2264k      0 --:--:-- --:--:-- --:--:-- 2265k
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.6.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xtrans/xtrans_1.6.0_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39892  100 39892    0     0   837k      0 --:--:-- --:--:-- --:--:--  846k
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.12 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libx11/libx11_1.8.12_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 1867k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1867k  100 1867k    0     0  4887k      0 --:--:-- --:--:-- --:--:-- 4887k
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.12 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxrender/libxrender_0.9.12_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 24568  100 24568    0     0  66486      0 --:--:-- --:--:-- --:--:-- 66579
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.9 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxft/libxft_2.3.9_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 47884  100 47884    0     0   972k      0 --:--:-- --:--:-- --:--:--  974k
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxext/libxext_1.3.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 97968  100 97968    0     0  2032k      0 --:--:-- --:--:-- --:--:-- 2079k
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxss/libxss_1.2.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 11924  100 11924    0     0   261k      0 --:--:-- --:--:-- --:--:--  264k
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/tk/tk_8.6.14_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1715k  100 1715k    0     0  30.9M      0 --:--:-- --:--:-- --:--:-- 31.6M
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.12.10 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python/python_3.12.10_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 7365k  100 7365k    0     0  88.5M      0 --:--:-- --:--:-- --:--:-- 88.7M
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.12.10 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-tkinter/python-tkinter_3.12.10_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1188k  100 1188k    0     0  23.7M      0 --:--:-- --:--:-- --:--:-- 24.1M
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.12.10 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-ensurepip-wheels/python-ensurepip-wheels_3.12.10_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1759k  100 1759k    0     0  33.3M      0 --:--:-- --:--:-- --:--:-- 33.6M
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency libxml2@2.14.3 if necessary...
Found libxml2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2/libxml2_2.14.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  442k  100  442k    0     0  9358k      0 --:--:-- --:--:-- --:--:-- 9408k
extracting libxml2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-utils@2.14.3 if necessary...
Found libxml2-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-utils/libxml2-utils_2.14.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 33996  100 33996    0     0   882k      0 --:--:-- --:--:-- --:--:--  897k
extracting libxml2-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python@2.14.3 if necessary...
Found libxml2-python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python/libxml2-python_2.14.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  171k  100  171k    0     0  3961k      0 --:--:-- --:--:-- --:--:-- 3982k
extracting libxml2-python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python-static@2.14.3 if necessary...
Found libxml2-python-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python-static/libxml2-python-static_2.14.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 63760  100 63760    0     0  1123k      0 --:--:-- --:--:-- --:--:-- 1132k
extracting libxml2-python-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland@1.23.1-2 if necessary...
Found libwayland in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland/libwayland_1.23.1-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  119k  100  119k    0     0  1961k      0 --:--:-- --:--:-- --:--:-- 1993k
extracting libwayland to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-cross-scanner@1.23.1-2 if necessary...
Found libwayland-cross-scanner in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland-cross-scanner/libwayland-cross-scanner_1.23.1-2_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 84860  100 84860    0     0  1821k      0 --:--:-- --:--:-- --:--:-- 1841k
extracting libwayland-cross-scanner to /home/builder/.termux-build/_cache-all...
Downloading dependency libxrandr@1.5.4 if necessary...
Found libxrandr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxrandr/libxrandr_1.5.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 21824  100 21824    0     0   444k      0 --:--:-- --:--:-- --:--:--  453k
extracting libxrandr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency spirv-headers@1.4.313.0 if necessary...
Found spirv-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-headers/spirv-headers_1.4.313.0_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  144k  100  144k    0     0   200k      0 --:--:-- --:--:-- --:--:--  200k
extracting spirv-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency spirv-tools@1.4.313.0 if necessary...
Found spirv-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-tools/spirv-tools_1.4.313.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  1 1842k    1 32427    0     0  64675      0  0:00:29 --:--:--  0:00:29 64595
100 1842k  100 1842k    0     0  1546k      0  0:00:01  0:00:01 --:--:-- 1547k
extracting spirv-tools to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency vulkan-headers@1.4.316 if necessary...
Found vulkan-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-headers/vulkan-headers_1.4.316_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1438k  100 1438k    0     0  26.7M      0 --:--:-- --:--:-- --:--:-- 27.0M
extracting vulkan-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency vulkan-utility-libraries@1.4.316 if necessary...
Found vulkan-utility-libraries in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-utility-libraries/vulkan-utility-libraries_1.4.316_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 13  523k   13 73728    0     0  85149      0  0:00:06 --:--:--  0:00:06 85136
100  523k  100  523k    0     0   600k      0 --:--:-- --:--:-- --:--:--  600k
extracting vulkan-utility-libraries to /home/builder/.termux-build/_cache-aarch64...
Downloading vulkan-validation-layers@1.4.316 source from 'https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.4.316.tar.gz' if necessary...
Downloading https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.4.316.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 3584k    0 3584k    0     0  5877k      0 --:--:-- --:--:-- --:--:-- 5877k
100 6229k    0 6229k    0     0  6036k      0 --:--:--  0:00:01 --:--:-- 6268k
Applying patch: 0001-no-android.patch
Applying patch: 0002-getenv.patch
Applying patch: 0003-layers-utils-android_ndk_types.h.patch
Applying patch: 0004-setenv.patch
Downloading https://github.com/Kitware/CMake/releases/download/v3.31.4/cmake-3.31.4-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

 91 52.4M   91 47.8M    0     0   153M      0 --:--:-- --:--:-- --:--:--  153M
100 52.4M  100 52.4M    0     0   159M      0 --:--:-- --:--:-- --:--:--  271M
Downloading https://github.com/ninja-build/ninja/releases/download/v1.12.1/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  120k  100  120k    0     0   800k      0 --:--:-- --:--:-- --:--:--  800k
Archive:  /home/builder/.termux-build/vulkan-validation-layers/tmp/ninja-1.12.1.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.12.1/ninja  
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The CXX compiler identification is Clang 18.0.3
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using parallel_hashmap as custom hash maps
-- Configuring done (0.3s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_C_FLAGS
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/vulkan-validation-layers/build
[1/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/error_location.cpp.o
[2/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/error_location_helper.cpp.o
[3/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/pnext_chain_extraction.cpp.o
[4/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/feature_requirements_helper.cpp.o
[5/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_object_types.cpp.o
[6/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/logging.cpp.o
[7/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/convert_utils.cpp.o
[8/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/dispatch_utils.cpp.o
[9/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/file_system_utils.cpp.o
[10/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_dispatch_table_helper.cpp.o
[11/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/hash_util.cpp.o
[12/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_extension_helper.cpp.o
[13/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/image_layout_utils.cpp.o
[14/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/keyboard.cpp.o
[15/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/image_utils.cpp.o
[16/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_layer_extension_utils.cpp.o
[17/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_struct_compare.cpp.o
[18/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/text_utils.cpp.o
[19/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vk_layer_config.cpp.o
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vk_layer_config.cpp:46:13: warning: unused function 'PropCallback' [-Wunused-function]
   46 | static void PropCallback(void *cookie, [[maybe_unused]] const char *name, const char *value, [[maybe_unused]] uint32_t serial) {
      |             ^~~~~~~~~~~~
1 warning generated.
[20/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/ray_tracing_utils.cpp.o
[21/195] Building CXX object layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o
FAILED: layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27c-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/sysroot -DUSE_CUSTOM_HASH_MAP -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/external -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o -MF layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o.d -o layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_indexing_oob_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_indexing_oob_pass.cpp:18:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/module.h:24:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/type_manager.h:22:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:266:19: error: no member named 'OpTensorReadARM' in namespace 'spv'
  266 |         case spv::OpTensorReadARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:267:19: error: no member named 'OpTensorQuerySizeARM' in namespace 'spv'
  267 |         case spv::OpTensorQuerySizeARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:692:19: error: no member named 'OpTypeTensorARM' in namespace 'spv'
  692 |         case spv::OpTypeTensorARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:693:19: error: no member named 'OpTensorReadARM' in namespace 'spv'
  693 |         case spv::OpTensorReadARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:694:19: error: no member named 'OpTensorQuerySizeARM' in namespace 'spv'
  694 |         case spv::OpTensorQuerySizeARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:1299:19: error: no member named 'OpTypeTensorARM' in namespace 'spv'
 1299 |         case spv::OpTypeTensorARM:
      |              ~~~~~^
6 errors generated.
[22/195] Building CXX object layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o
FAILED: layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27c-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/sysroot -DUSE_CUSTOM_HASH_MAP -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/external -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o -MF layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o.d -o layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_class_general_buffer_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_class_general_buffer_pass.cpp:17:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:266:19: error: no member named 'OpTensorReadARM' in namespace 'spv'
  266 |         case spv::OpTensorReadARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:267:19: error: no member named 'OpTensorQuerySizeARM' in namespace 'spv'
  267 |         case spv::OpTensorQuerySizeARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:692:19: error: no member named 'OpTypeTensorARM' in namespace 'spv'
  692 |         case spv::OpTypeTensorARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:693:19: error: no member named 'OpTensorReadARM' in namespace 'spv'
  693 |         case spv::OpTensorReadARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:694:19: error: no member named 'OpTensorQuerySizeARM' in namespace 'spv'
  694 |         case spv::OpTensorQuerySizeARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:1299:19: error: no member named 'OpTypeTensorARM' in namespace 'spv'
 1299 |         case spv::OpTypeTensorARM:
      |              ~~~~~^
6 errors generated.
[23/195] Building CXX object layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o
FAILED: layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27c-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/sysroot -DUSE_CUSTOM_HASH_MAP -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/external -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o -MF layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o.d -o layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_class_texel_buffer_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_class_texel_buffer_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/module.h:24:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/type_manager.h:22:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:266:19: error: no member named 'OpTensorReadARM' in namespace 'spv'
  266 |         case spv::OpTensorReadARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:267:19: error: no member named 'OpTensorQuerySizeARM' in namespace 'spv'
  267 |         case spv::OpTensorQuerySizeARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:692:19: error: no member named 'OpTypeTensorARM' in namespace 'spv'
  692 |         case spv::OpTypeTensorARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:693:19: error: no member named 'OpTensorReadARM' in namespace 'spv'
  693 |         case spv::OpTensorReadARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:694:19: error: no member named 'OpTensorQuerySizeARM' in namespace 'spv'
  694 |         case spv::OpTensorQuerySizeARM:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:1299:19: error: no member named 'OpTypeTensorARM' in namespace 'spv'
 1299 |         case spv::OpTypeTensorARM:
      |              ~~~~~^
6 errors generated.
[24/195] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_function_pointers.cpp.o
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2025-05-31 06:09:49 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/15360800010">15360800010</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,11 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.4.315"
+TERMUX_PKG_VERSION="1.4.319"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=9e85fcd793fb1ee871ea8e293cb1f9736473614be23dd03a0dec8a5d1cee918c
+TERMUX_PKG_SHA256=e46cb2ba7190cd134497c9841c967e39b8e5f45d4c1cd85f0d3a827e8b4bf037
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
-TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
+TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+\.\d+"
@@ -20,4 +20,51 @@ termux_pkg_auto_update() {
 	fi
 	local latest_version=$(echo "${latest_refs_tags}" | sort -V | tail -n1)
 	termux_pkg_upgrade_version "${latest_version}"
+}
+
+termux_step_pre_configure() {
+	termux_setup_cmake
+	termux_setup_ninja
+
+	# use upstream known_good.json to fetch and build deps
+	"${TERMUX_PKG_SRCDIR}/scripts/update_deps.py" --generator Ninja
+
+	find "$PWD" -name helper.cmake | sort | xargs -i bash -c "echo '===== {} =====' && cat {}"
+
+	# prioritises build deps from source instead of system prefix
+	# avoiding pollution from old dependencies
+	local GLSLANG_INSTALL_DIR="${TERMUX_PKG_SRCDIR}/glslang/build/install"
+	local GOOGLETEST_INSTALL_DIR="${TERMUX_PKG_SRCDIR}/googletest/build/install"
+	local SPIRV_HEADERS_INSTALL_DIR="${TERMUX_PKG_SRCDIR}/SPIRV-Headers/build/install"
+	local SPIRV_TOOLS_INSTALL_DIR="${TERMUX_PKG_SRCDIR}/SPIRV-Tools/build/install"
+	local VULKAN_HEADERS_INSTALL_DIR="${TERMUX_PKG_SRCDIR}/Vulkan-Headers/build/install"
+	local VULKAN_UTILITY_LIBRARIES_INSTALL_DIR="${TERMUX_PKG_SRCDIR}/Vulkan-Utility-Libraries/build/install"
+
+	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+="
+	-DGLSLANG_INSTALL_DIR=${GLSLANG_INSTALL_DIR}
+	-DGOOGLETEST_INSTALL_DIR=${GOOGLETEST_INSTALL_DIR}
+	-DSPIRV_HEADERS_INSTALL_DIR=${SPIRV_HEADERS_INSTALL_DIR}
+	-DSPIRV_TOOLS_INSTALL_DIR=${SPIRV_TOOLS_INSTALL_DIR}
+	-DVULKAN_HEADERS_INSTALL_DIR=${VULKAN_HEADERS_INSTALL_DIR}
+	-DVULKAN_UTILITY_LIBRARIES_INSTALL_DIR=${VULKAN_UTILITY_LIBRARIES_INSTALL_DIR}
+	"
+
+	CFLAGS="-I${GLSLANG_INSTALL_DIR}/include ${CFLAGS}"
+	CFLAGS="-I${GOOGLETEST_INSTALL_DIR}/include ${CFLAGS}"
+	CFLAGS="-I${SPIRV_HEADERS_INSTALL_DIR}/include ${CFLAGS}"
+	CFLAGS="-I${SPIRV_TOOLS_INSTALL_DIR}/include ${CFLAGS}"
+	CFLAGS="-I${VULKAN_HEADERS_INSTALL_DIR}/include ${CFLAGS}"
+	CFLAGS="-I${VULKAN_UTILITY_LIBRARIES_INSTALL_DIR}/include ${CFLAGS}"
+	CXXFLAGS="-I${GLSLANG_INSTALL_DIR}/include ${CXXFLAGS}"
+	CXXFLAGS="-I${GOOGLETEST_INSTALL_DIR}/include ${CXXFLAGS}"
+	CXXFLAGS="-I${SPIRV_HEADERS_INSTALL_DIR}/include ${CXXFLAGS}"
+	CXXFLAGS="-I${SPIRV_TOOLS_INSTALL_DIR}/include ${CXXFLAGS}"
+	CXXFLAGS="-I${VULKAN_HEADERS_INSTALL_DIR}/include ${CXXFLAGS}"
+	CXXFLAGS="-I${VULKAN_UTILITY_LIBRARIES_INSTALL_DIR}/include ${CXXFLAGS}"
+	LDFLAGS="-L${GLSLANG_INSTALL_DIR}/lib ${LDFLAGS}"
+	LDFLAGS="-L${GOOGLETEST_INSTALL_DIR}/lib ${LDFLAGS}"
+	LDFLAGS="-L${SPIRV_HEADERS_INSTALL_DIR}/lib ${LDFLAGS}"
+	LDFLAGS="-L${SPIRV_TOOLS_INSTALL_DIR}/lib ${LDFLAGS}"
+	LDFLAGS="-L${VULKAN_HEADERS_INSTALL_DIR}/lib ${LDFLAGS}"
+	LDFLAGS="-L${VULKAN_UTILITY_LIBRARIES_INSTALL_DIR}/lib ${LDFLAGS}"
 }

